tests/conftest.py

```diff
@@ -158,3 +163,28 @@ def override_get_qc_db():
     client = TestClient(app)
 
     return client
+
+
+def insert_from_yaml(session, dir_path, module_name):
```

Comment:
Maybe there is a "fixtures from yaml" package out there somewhere?

Reply:
Well, there is. For per model testing. Intimately related to Flask. I am not sure we wil get much value